go.opentelemetry.io/vanity.yaml

```diff
@@ -1,7 +1,11 @@
 host: go.opentelemetry.io
 
 paths:
-  /otel:
+  /lib:
     repo: https://github.com/open-telemetry/opentelemetry-go
   /collector:
     repo: https://github.com/open-telemetry/opentelemetry-collector
+  /contrib/lib:
```

Comment:
I would prefer to not use `lib` in our paths.

Reply:
contrib cannot be used because collector has also contrib.